modules/govuk/manifests/apps/ckan.pp

```diff
@@ -0,0 +1,103 @@
+# == Class: govuk::apps::ckan
+##
+# === Parameters
+#
+# [*enabled*]
+#   Should the app exist?
+#
+# [*port*]
+#   What port should the app run on?
+#
+# [*db_hostname*]
+#   The postgres instance for CKAN to connect to
+#
+# [*db_username*]
+#   The username for the postgresql role
+#
+# [*db_password*]
+#   The password for the postgresql role
+#
+# [*db_name*]
+#   The database that CKAN should use
+#
+# [*redis_host*]
+#   The Redis host that CKAN should use
+#
+# [*redis_port*]
+#   The Redis port that CKAN should use
+#
+# [*secret*]
+#   The secret token that CKAN uses for session encryption
+#
+class govuk::apps::ckan (
+  $enabled       = false,
+  $port          = '3108',
+  $db_hostname   = undef,
+  $db_username   = 'ckan',
+  $db_password   = 'foo',
+  $db_name       = 'ckan_production',
+  $redis_host    = undef,
+  $redis_port    = '6379',
+  $secret        = undef,
+  $ckan_site_url = undef,
+) {
+  $ckan_home = '/var/ckan'
+  $ckan_ini  = "${ckan_home}/ckan.ini"
+
+  if $enabled {
+    govuk::app { 'ckan':
+      app_type           => 'bare',
+      command            => "unicornherder --gunicorn-bin ./venv/bin/gunicorn -p /var/run/ckan/unicornherder.pid -- --pythonpath ${ckan_home} wsgi_app:application --bind 127.0.0.1:3108 --workers 8",
+      port               => $port,
+      vhost_ssl_only     => true,
+      health_check_path  => '/',
```

Comment:
We've created a `/healthcheck` path for this. See https://github.com/alphagov/ckanext-datagovuk/pull/54.